hibernate-core/src/main/java/org/hibernate/binder/internal/TenantIdBinder.java

```diff
@@ -59,7 +59,8 @@ public void bind(
 							"",
 							singletonMap( PARAMETER_NAME, tenantIdType ),
 							Collections.emptyMap(),
-							false
+							true,
```

Comment:
The tenant id filter is automatically enabled when a non-root tenant id is set.

```suggestion
							false,
```

Reply:
Yeah, makes sense, since it causes the errors on the root tenant execution (since the param is not set)